runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultConfig.java

```diff
@@ -42,6 +42,16 @@ default boolean showTimer()
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "secondsTimer",
+		name = "Show elapsed seconds",
+		description = "Instead of showing the call change timer, the wave elapsed time will be shown (in seconds)"
```

Comment:
This description needs to be changed now that you no longer remove the other timer

Reply:
Ah yea good point